hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -948,6 +948,8 @@
       'lc_ctrl.lc_iso_part_sw_wr_en'       : ['flash_ctrl.lc_iso_part_sw_wr_en'],
       'lc_ctrl.lc_seed_hw_rd_en'           : ['otp_ctrl.lc_seed_hw_rd_en',
                                               'flash_ctrl.lc_seed_hw_rd_en'],
+
+      //'spi_device.passthrouth': ['spi_host0.passthrough']
```

Comment:
Typo?

Do I understand correctly that this would still have to be integrated with SPI Host?

Reply:
Yep. Typo. Fixed.